Makefile

```diff
@@ -1,9 +1,10 @@
 format:
-	isort --recursive src
```

Comment:
I guess I'll finally upgrade to Fedora 33 from F31, where there is no isort 5.  :)

Reply:
ohhh :D sorry maybe its time as the 31 is already "stable"